mava/wrappers/jaxmarl.py

```diff
@@ -241,29 +235,19 @@ def step(
     def observation_spec(self) -> specs.Spec:
         agents_view = jaxmarl_space_to_jumanji_spec(merge_space(self._env.observation_spaces))
         single_agent_action_space = self._env.action_space(self.agents[0])
-        # we can't mask continuous actions, so just return a shape of 0 for this
-        n_actions = (
-            single_agent_action_space.n
-            if _is_discrete(self._env.action_space(self._env.agents[0]))
-            else 0
-        )
+
+        # we can't mask continuous actions, so we return a mask of shape 0.
+        n_actions = single_agent_action_space.n if _is_discrete(single_agent_action_space) else 0
```

Comment:
Could you also make this an abstract property, then we don't need to do this if as we know smax is discrete and mabrax is continuous

Reply:
Also looks like this causes a size mismatch for mabrax as you have a action_mask shape of 0 in the spec but you return a different shape for the action mask in mabrax, so making it a property should help avoid this